### DISCUSSION TITLE
Only send latest RPC to server for 'client auth' syncvars.

### DISCUSSION THREAD

**Discussion by FirstGearGames** [FirstGearGames/FishNet#286] [Feature Request]
Currently when a client sets a syncvar using properties it's done by sending a RPC to the server, and the server then changes the value. However, the client may change a value several times per tick resulting in a lot of necessary RPC sends, as only the latest really matters.

Make it so when RPCs are used as syncvar properties to only send the latest value to the server per INTERVAL.

**Comment by FirstGearGames** [comment]
This idea is getting scrapped in favor of client-authoritative synctypes.

**Reply by funnymanwin** [reply]
Hi. I don't understand what's going on. I developed a project on v3.5 and now it's 4.5 and i used `[SyncVar]` attribute, but now in documentation there is no information about `[SyncVar]`. Now it says to use new `SyncVar<>` type. It's okay, but there said again that:
> When modifying SyncVars using client-side, a RPC is sent with every set. If your SyncVar value will change frequently consider limiting how often you set the value client-side to reduce bandwidth usage.

So as I understand, there is still no solution to make client-authoritative property without synchronization per every set operation?
